### PR TITLE
Add wgrib2 and ip2 libraries

### PR DESCRIPTION
**Pull request by kgerheiser**

### CODE REVIEW

**Comment by kgerheiser** [comment]
Modules, forgot about those.

**Comment by aerorahul** [comment]
> Modules, forgot about those.

for `ip2`, you can look at `ip` and copy paste from there.
for `wgrib2`, you can take `nemsio` as an example or any other one that you think resembles `wgrib2`.  Are you also building the application with the library? If so, `PATH` should be appended.

**Comment by kgerheiser** [comment]
We've got modules

**Comment by kgerheiser** [comment]
Could you take a look at this again @aerorahul. I added the modules

File: COMPONENTS [hunk 1]
```diff
@@ -18,4 +19,5 @@ crtm        | v2.3.0            | 2.3.0
 nceppost    | dceca26           | dceca26
 wrf_io      | v1.1.1-cmake      | 1.1.1
 bufr        | bufr_v11.3.1.1    | 11.3.1.1
+wgrib2      | feature/cmake     | 2.0.8
```
Comment (aerorahul) [inline COMPONENTS]: Do you want to create a branch say `v2.0.8-cmake` instead of `feature/cmake`

**Review by aerorahul — APPROVED** [review]
Everything looks good, with the exception of a floating branch `feature/cmake` since you are tagging it with a version.

File: COMPONENTS (hunk 1 above)
Comment (kgerheiser) [reply COMPONENTS]: Yes, I will do that

Comment (aerorahul) [reply COMPONENTS]: you can merge after that change. thanks!

**Pull request merged**